napari/_lazy.py

```diff
@@ -1,5 +1,7 @@
 from importlib import import_module
 
+import napari._qt.qt_main_window
```

Comment:
this will break the point of the lazy imports, and also make it so that napari can't be used without having qtpy installed (it can! ... see the [headless test fail](https://github.com/napari/napari/pull/3149/checks?check_run_id=3312609895))